src/redux/actions/appointmentAction.ts

```diff
@@ -20,10 +20,16 @@ export interface IAppointmentActionSetAppointments
   payload: { appointments: IAppointment[] };
 }
 
+export interface IAppointmentActionCreateAppointment 
+  extends IAppointmentActionBase {
+  payload: { service_type: string; start_time: string; end_time: string}
```

Comment:
Lets update to `serviceType`, `startTime` and `endTime` as I previously commented. Map to the api keys in the saga when constructing your request